src/helpers/snapshot.ts

```diff
@@ -46,38 +40,40 @@ export async function getProposal(space, id) {
 
 export async function getResults(space, proposal, votes) {
   try {
-    const provider = getProvider(space.network);
     const voters = votes.map(vote => vote.voter);
+    const provider = getProvider(space.network);
     const strategies = proposal.strategies ?? space.strategies;
     /* Get scores */
-    console.time('getProposal.scores');
-    const [scores, profiles]: any = await Promise.all([
-      getScores(
-        space.key,
-        strategies,
-        space.network,
-        provider,
-        voters,
-        parseInt(proposal.snapshot)
-      ),
-      getProfiles([proposal.author, ...voters])
-    ]);
-    console.timeEnd('getProposal.scores');
-    console.log('Scores', scores);
+    if (proposal.state !== 'pending') {
+      console.time('getProposal.scores');
+      const [scores, profiles]: any = await Promise.all([
+        getScores(
+          space.key,
+          strategies,
+          space.network,
+          provider,
+          voters,
+          parseInt(proposal.snapshot)
+        ),
+        getProfiles([proposal.author, ...voters])
```

Comment:
We wont get the profile of the proposal author when it's pending.

Reply:
But that's ok we can solve this later with https://trello.com/c/wU0PrcRe/165-create-composable-to-load-profiles